site/content/resources/bay-area/Blackbox.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Blackbox
```

Comment:
For the file name, the title should be capitalized but the markdown file name shouldn't be to follow naming conventions. Ergo, the files should be made lowercase to be `site/content/resources/bay-area/blackbox.md` instead of `site/content/resources/bay-area/Blackbox.md`.

Reply:
Thanks for pointing this out! I'll go ahead and update the markdown files accordingly moving forward.